docs/sinks/oracle.md

```diff
@@ -7,8 +7,9 @@ description: Replicate data from your sources to Oracle Database.
 
 ### Configuration
 
+To start, define a new connection to Oracle in your `dozer-config.yaml` file using the steps outlined [here](../sources/oracle.md).
```

Comment:
Try using something more descriptive than `here` for your link. Tell the user where it's going to take them, like
> using the steps outlined in the [Oracle connection configuration section](../sources/oracle.md)